moxy/org.eclipse.persistence.moxy/src/main/java/org/eclipse/persistence/jaxb/plugins/BeanValidationPlugin.java

```diff
@@ -209,7 +206,7 @@ public int parseArgument(Options opt, String[] args, int i) throws BadCommandLin
 
     /* ######### CORE FUNCTIONALITY ######### */
     private static final String PATTERN_ANNOTATION_NOT_APPLICABLE = "Facet \"pattern\" was detected on a DOM node with non-string base type. Annotation was not generated, because it is not supported by the Bean Validation specification.";
-    private final boolean securityEnabled = System.getSecurityManager() != null;
+    //private final boolean securityEnabled = System.getSecurityManager() != null;
```

Comment:
remove?

Reply:
Sure. :) Just keep in mind that it's 'in progress' quality changeset.

Reply:
Removed